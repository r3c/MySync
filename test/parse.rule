invalid line
missing: no command placeholder
$: invalid name character
dummy: echo {}
    invalid=50
    interval=a
    keep=b
