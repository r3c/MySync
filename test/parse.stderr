^error: ignoring unrecognized line 'invalid line' in file '[^']*/parse\.rule' at line #1$
^error: command 'no command placeholder' does not contain a placeholder for rule 'missing' in file '[^']*/parse\.rule' at line #2$
^error: ignoring unrecognized line '\$: invalid name character' in file '[^']*/parse\.rule' at line #3$
^warning: option 'invalid' is unknown in file '[^']*/parse\.rule' at line #5$
^warning: option 'interval' has a non-integer value 'a' for rule 'dummy' in file '[^']*/parse\.rule' at line #6$
^warning: option 'keep' has a non-integer value 'b' for rule 'dummy' in file '[^']*/parse\.rule' at line #7$
^dummy: should backup with \[interval=86400\]\[keep=7\]$
^$
